sympy/physics/optics/medium.py

```diff
@@ -5,11 +5,18 @@
 """
 
 from __future__ import division
+from sympy.physics.units import second, meter, kilogram, ampere
 
 __all__ = ['Medium']
 
 from sympy import Symbol, sympify, sqrt
-from sympy.physics.units import c, u0, e0
+from sympy.physics.units import speed_of_light, u0, e0
+
+
+c = speed_of_light.convert_to(meter/second)
+_e0mksa = e0.convert_to(ampere**2*second**4/(kilogram*meter**3))
+_u0mksa = u0.convert_to(meter*kilogram/(ampere**2*second**2))
```

Comment:
@asmeurer these are the only adaptations that I made to use the new units module in place of the old. Note that the units behave exactly the same, despite now having a completely different definitions.

The API is otherwise identical, the optics module tests work without edits with the new units module.